recipes-utils/ultra96-radio-leds/files/ultra96-radio-leds.sh

```diff
@@ -11,11 +11,31 @@
 ### END INIT INFO
 
 SYS_GPIO_FOLDER=/sys/class/gpio
-WIFI_LED=506
-BT_LED=505
+WIFI_LED=498
+BT_LED=497
 LED_ON=1
 LED_OFF=0
 
+for gpiochip in `ls /sys/class/gpio | grep gpiochip`
+do
+	label=$(cat /sys/class/gpio/$gpiochip/label)
+	base=$(cat /sys/class/gpio/$gpiochip/base)
```

Comment:
This could be moved inside `if [[ "$label" == *"a0050000.gpio"* ]]; then`

Reply:
I don't understand this request.
I pushed a new version to the branch with your review feedback.